cli/src/cli.rs

```diff
@@ -416,11 +427,22 @@ pub async fn init_command(
         .map(|addr| Bech32Address::from_str(&addr))
         .transpose()?;
 
-    Ok(Wallet::builder()
+    let public_key_options = init_params.bip_path.map(|bip| {
+        LedgerOptions::new(
+            PublicKeyOptions::new(bip.coin_type)
+                .with_account_index(bip.account)
+                .with_internal(bip.change != 0)
+                .with_address_index(bip.address_index),
```

Comment:
Can now just be `PublicKeyOptions::from(bip)`, also in more places

Reply:
ah, good point, thanks